openstudiocore/src/osversion/VersionTranslator.cpp

```diff
@@ -2638,18 +2639,15 @@ std::string VersionTranslator::update_1_8_3_to_1_8_4(const IdfFile& idf_1_8_3, c
       m_refactored.push_back( std::pair<IdfObject,IdfObject>(object,newObject) );
       ss << newObject;
     } else if(iddname == "OS:AvailabilityManager:Scheduled") {
-      m_untranslated.push_back(object);
-      continue;
+      m_deprecated.push_back(object);
```

Comment:
No I probably should have made two issues. The above fixes 1955 - OSVersionFixture.VersionTranslator_ExampleModel_1_1 (Failed) a legitimate thing.

The rubbish default tests are failing because of [this assertion](https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/osversion/VersionTranslator.cpp#L2604) which depends on a required relationship to another object. Naturally these required relationships are not established in the default test methodology so the assertion fails. What is surprising is that we went this far without uncovering a problem with this tests. So it is.